src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,76 @@
+.footer {
+  &__title {
+    color: #1b2129;
```

Comment:
You are inlining this color multiple times
Please take it to the variables

Also check other colors